collection-scripts/gather

```diff
@@ -197,6 +204,12 @@ if [ "$clusterscoped" == true ] && [ "$odf" == false ]; then
     pids+=($!)
 fi
 
+if [ "$minimal_crds" == true ] && [ "$odf" == false ]; then
```

Comment:
```suggestion
if [ "$minimal_crds" == true ]; then
```
We do not need to check for `odf == false` here as the minimal script is not part of the odf collection. The reason we check for `odf == false` is so we do not collect resources that are collected with `--odf` mode twice.

Reply:
Umm.. I think you forgot to commit the changes..

P.S: We should not mark conversations as resolved by ourselves as it makes it harder to keep track of changes for reviewers. Ideally, the person who left the comment resolves it upon confirming the changes :)

Reply:
Noted, will be careful with future PRs.
Pushed the correct changes